miso-web/src/main/webapp/scripts/pane_project.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+ * MISO project contacts: Robert Davey @ TGAC
+ * *********************************************************************
+ *
+ * This file is part of MISO.
+ *
+ * MISO is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MISO is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MISO.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * *********************************************************************
+ */
+
+PaneTarget.project = (function() {
+  var title = "Project";
+  var url = "/miso/rest/project/search";
+
+  var transform = function(project) {
+    return Tile.make([Tile.title(project.shortName || project.alias),
+        Tile.lines(["Name: " + project.name, project.shortName ? ("Alias: " + project.alias) : null].filter(function(x) {
```

Comment:
Is this ternary correct? If shortName is falsey, use `null` ?

Reply:
Yes. If `shortName` is falsey, then `alias` is already used for the tile title, so doesn't need to be included in the details